app/initializers/irene.js

```diff
@@ -1,25 +1,91 @@
 import ENV from 'irene/config/environment';
 
-const runtimeConfig = window.runtimeGlobalConfig;
+class ENVHandler {
+  constructor(envHandlerConst) {
+    this.envHandlerConst = envHandlerConst;
+  }
 
-function isTrue(value) {
-  value = String(value).toLowerCase();
-  return value === 'true';
-}
+  isRuntimeAvailable() {
+    return !(typeof runtimeGlobalConfig == "undefined");
```

Comment:
how does this variable becomes accessible?

Reply:
runtimeGlobalConfig is injected via a script as global variable. See runtimeconfig.js